### DISCUSSION TITLE
The time-detonator inside the atomic bomb does not detonate the atomic bomb. Even with C-4 explosion

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15129] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I started the timer in the detonator and put c-4 in it. Then I put this detonator in the deep nuclear decoy.
I threw the nuclear decoy out through the lower airlock.
The c-4 exploded, but the decoy did not explode.
The atomic decoy remained intact. The time-detonator and c-4 have gone. Probably the detonator exploded along with the c-4.
I checked the same with the atomic bomb. It did not explode too.
Here I was promised that a nuclear bomb can be detonated using a detonator and c-4.
https://github.com/FakeFishGames/Barotrauma/discussions/13690
Perhaps the detonator with c-4 in should not be in the bomb, but nearby?
But it seems to me that the detonator should detonate the bomb if the detonator explodes inside the bomb. Then you can use bombs even without a launch tube.
Second moment: I discovered that the atomic bomb falls faster than the atomic decoy if you throw it out through the lower airlock. 
Is this how it was intended?

### Reproduction steps

1. start the timer in the detonator (set delay to 20 second or more) and put c-4 in it. 
2. Then put this detonator in the nuclear decoy or nuclear charge.
3. threw the nuclear bomb/decoy out through the lower airlock.
4. Time passed and the c-4 exploded, but the decoy/bomb did not explode.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
On a closer look at the xml, this seems to be intentional: the nuclear explosives need to be "armed" by firing them first, otherwise they're unaffected by explosions.
